src/arith/int_set.cc

```diff
@@ -694,6 +772,18 @@ IntSet EvalSet(Range r, const std::unordered_map<const VarNode*, IntSet>& dom_ma
   return EvalSet(r, ConvertDomMap(dom_map));
 }
 
+Array<IntSet> EvalSet(const Array<Range>& region, const Map<Var, IntSet>& dom_map) {
+  Analyzer ana;
+  IntervalSetEvaluator m(&ana, dom_map);
+  Array<IntSet> result;
+  result.reserve(region.size());
+  for (const Range& r : region) {
+    PrimExpr sum = r->min + r->extent - 1;
```

Comment:
r->min + (r->extent -1)

Reply:
This will allow constant fold of r->extent -1 and avoid the use of analyzer simplifcation